spec/integration/categories_index_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'Categories', type: :system do
+  describe 'index' do
+    before :all do
+      @user = User.new(name: 'nameofuser', email: 'asdt56062@gmail.com', password: '6letters',
+                       encrypted_password: '$2a$12$192AtELpNZ0aZCfnSxs35umQYmbSn52FK8ML/vY.iZvDW4FvkvHn2')
+      @user.skip_confirmation!
+      @user.confirm
+      @user.save
+      sign_in @user
```

Comment:
- [ ] There are some tests not passing, Please fix that.

![image](https://user-images.githubusercontent.com/68796232/219522039-c6d342c1-e928-4dae-bf5d-0b7f2957e44a.png)